keras_nlp/benchmarks/sentiment_analysis.py

```diff
@@ -0,0 +1,143 @@
+# Copyright 2023 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import inspect
+import time
+
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from absl import app
+from absl import flags
+from tensorflow import keras
+
+import keras_nlp
+
+# Use mixed precision for optimal performance
+keras.mixed_precision.set_global_policy("mixed_float16")
+
+FLAGS = flags.FLAGS
+flags.DEFINE_string(
+    "model",
+    None,
+    "The name of the classifier such as BertClassifier.",
+)
+flags.DEFINE_string(
+    "preset",
+    None,
+    "The name of a preset, e.g. bert_base_multi.",
+)
+
+flags.DEFINE_float("learning_rate", 5e-5, "The learning rate.")
+flags.DEFINE_integer("num_epochs", 1, "The number of epochs.")
+flags.DEFINE_integer("batch_size", 16, "The batch size.")
+
+tfds.disable_progress_bar()
+
+BUFFER_SIZE = 10000
+
+
+def check_flags():
+    if not FLAGS.model:
```

Comment:
I just remembered there is actually a way to do this with absl directly. `flags.mark_flag_as_required("flag")`.

https://github.com/keras-team/keras-nlp/blob/master/examples/bert_pretraining/bert_pretrain.py#L454